File: Python/cryptography.py
import random
st=input("enter your string")
words=st.split(" ")
coding=True
if(coding):
    nword=[]
    for word in words:
        if(len(word)>=3):
            r1="dsf"
            r2="jkr"
            stnew=r1+word[1:]+word[0]+r2
            nword.append(stnew)
        else:
            nword.append(word[::-1])
    print(" ".join(nword))
else:
    nword=[]
    for word in words:
        if(len(word)>=3):
            r1="dsf"
            r2="jkr"
            stnew=r1-word[:-1]+word[0]-r2
            nword.append(stnew)
        else:
            nword.append(word[::-1])
    

File: c/cryptography.c
#include<stdio.h>
#include<string.h>
int main(){
    char string[20];
    printf("Enter your String:");
    scanf("%s",string);
    for(int i=0;i<strlen(string);i++){
        i%2==0?string[i]=string[i]++:string[i]--;
    }
    printf("%s",string);
}